rust/arrow/src/csv/writer.rs

```diff
@@ -105,12 +105,12 @@ pub struct Writer<W: Write> {
     beginning: bool,
 }
 
-impl<'a, 'b: 'a, W: Write + 'b> Writer<W>
-where
-    &'a W: Write,
-{
+impl<W: Write> Writer<W> {
     /// Create a new CsvWriter from a writable object, with default options
     pub fn new(writer: W) -> Self {
+        let delimiter = b',';
+        let mut builder = csv_crate::WriterBuilder::new();
+        let writer = builder.delimiter(delimiter).from_writer(writer);
         Writer {
             writer,
             delimiter: b',',
```

Comment:
would it be possible to reuse the `delimiter` variable here?

Reply:
done.